### PR TITLE
Packages The Module

### PR DESCRIPTION
**Pull request by DanielMwendwa**
Copied & edited RapidProTools setup.py file.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from setuptools import setup
+
+setup(
+    name="CodaV2PythonClient",
+    version="0.1.0",
```
Comment (as2388) [inline setup.py:5]: I would start with 0.0.0 until we have a released version

**Review by as2388 — APPROVED** [review]
Once this is merged, please make sure you can install this into a test project, and successfully list the dataset ids from the coda staging instance.

File: setup.py (hunk 1 above)
Comment (DanielMwendwa) [reply setup.py]: Thanks Updated.

**Comment by DanielMwendwa** [comment]
> Once this is merged, please make sure you can install this into a test project, and successfully list the dataset ids from the coda staging instance.

Noted.

**Pull request merged**